cookbooks/cdo-apps/recipes/workers.rb

```diff
@@ -0,0 +1,36 @@
+# Set default number of worker processes based on reported CPU and memory.
+# These values can be overridden using override attributes in the Chef configuration,
+# but the defaults should usually be used (or tweaked/fixed).
+
+# Approximate max Dashboard memory usage per process
+DASHBOARD_USAGE = 800 * 1024
+
+# Approximate max Pegasus memory usage per process
+PEGASUS_USAGE = 400 * 1024
+
+# Ratio of Pegasus to Dashboard worker processes
+PEGASUS_DASHBOARD_RATIO = 0.5.to_f
+MEMORY_RATIO = 0.8.to_f
+
+varnish_storage_gb = node['cdo-varnish']['storage'].match(/malloc,(\d+[,.]?\d*)G/)
```

Comment:
added coverage for the other supported suffixes according to the [docs](https://www.varnish-cache.org/docs/trunk/users-guide/storage-backends.html#storage-backends).